CHANGELOG.md

```diff
@@ -10,6 +10,11 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 ## Unreleased
 
+- `code42 users list` command
```

Comment:
This should be under the `Added` header and be phrased like this

### Added

- New command `code42 users list` with options:
...